src/db.rs

```diff
@@ -25,7 +27,10 @@ impl DbLocker {
     const LOCK_KEY: i64 = 4036779288569897133;
 
     pub fn connect(config: &postgres::Config) -> anyhow::Result<Self> {
-        let mut pg = config.connect(NoTls)?;
+        let mut builder = SslConnector::builder(SslMethod::tls())?;
+        builder.set_verify(SslVerifyMode::NONE);
+        let connector = MakeTlsConnector::new(builder.build());
+        let mut pg = config.connect(connector)?;
```

Comment:
Will this break compatibility for anybody using Reshape with a non-TLS database already? I.e. will it refuse to connect if the database does not use TLS or will it work as usual?